packages/govuk-frontend/src/govuk/components/tabs/_index.scss

```diff
@@ -2,10 +2,14 @@
   .govuk-tabs {
     @include govuk-responsive-margin(1, "top");
     @include govuk-responsive-margin(6, "bottom");
+    @include govuk-font($size: 19);
```

Comment:
Our tab links `.govuk-tabs__tab` already use `@mixin govuk-link-common` so might not need this?

Can override the `font-size` only on `.govuk-tabs__tab` where we bump it up?

```suggestion
```

Reply:
I'm going to keep this if that's ok because without it, the heading and the list dashes on mobile don't get typography:

| With `govuk-font` | Without `govuk-font`|
| - | - |
| <img width="128" alt="Screenshot of tabs on mobile with typography" src="https://github.com/alphagov/govuk-frontend/assets/64783893/98c48a28-9050-4456-854a-2269dda9fd48"> |  <img width="145" alt="Screenshot of tabs on mobile without typography" src="https://github.com/alphagov/govuk-frontend/assets/64783893/d12aa668-f9d1-4eff-a927-bd2e7d42c012"> |

Once again, we've got an example where https://github.com/alphagov/govuk-frontend/issues/4277 could solve at least some of this.

Reply:
Sorry I should have joined the dots here

Same as https://github.com/alphagov/govuk-frontend/pull/4267#discussion_r1341203283 can we style only the elements without `font-family`?

* Add it to `.govuk-tabs__title` until we extend it with `.govuk-heading-s`
* Add it to `.govuk-tabs__list` until we extend it with `.govuk-list`

Happy to move to the component block though and document these instead?

Reply:
I'm trying to remember my sass compilation arithmetic here. Would moving it out of the top level mean we end up with more less generated CSS? In this case it's also stopover until we get to #4277 and also we're probably talking individual kilobytes so I'm happy to do this for now if you reckon it's better.

Reply:
Tell you what, with the amount of code that `govuk-font()` adds let's keep it at block level for now

Better to remove it entirely (later) when suggestions 1) and 2) can be picked up

Reply:
Thank you for compromising 😂